modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -104,7 +103,7 @@
 	alt_titles = list("Nanotrasen Representative", "Nanotrasen Diplomat", "Central Command Representative")
 
 /datum/job/orderly
-
+	alt_titles = list("Orderly", "Medical Attendant", "Ward Officer")
```

Comment:
Again I'd rather keep away from "officer".

Reply:
No additional titles for guards that aren't just their department + guard, ie. "Medical Guard"